controllers/controllers.js

```diff
@@ -64,4 +64,16 @@ exports.postNewComment = (req, res, next) => {
     .catch((err) => {
         next(err)
     })
+}
+
+exports.patchArticle = (req, res, next) => {
+    const newVote = req.body
+    const { article_id } = req.params
+    updateArticleVotes(newVote, article_id)
+    .then((article) => {
+        res.status(201).send(article)
```

Comment:
We should be sending back the article as an object with an appropriate key such as {article: article} or the shorthand would be {article}.